blog/rollkit-the-first-sovereign-rollup-framework.md

```diff
@@ -0,0 +1,74 @@
+---
+head:
+  - - meta
+    - name: title
+      content: "Rollkit: The First Sovereign Rollup Framework"
+  - - meta
+    - name: description
+      content: Launch a sovereign, customizable blockchain as easily as a smart contract
+  - - meta
+    - name: keywords
+      content: Rollkit, Cosmos-SDK
+  - - meta
+    - name: og:image
```

Comment:
since the image is set for the entire site, you don't really need to include this IMHO